gulp/constants/functional-test-globs.js

```diff
@@ -1,9 +1,9 @@
-const MULTIPLE_WINDOWS_TESTS_GLOB = 'test/functional/fixtures/multiple-windows/test.js';
-// NOTE: This source is specified temporary. I will change it to live mode test suit in the next PR.
-const HEADED_CHROME_FIREFOX_TESTS_GLOB = ['test/functional/fixtures/multiple-windows/test.js'];
-const COMPILER_SERVICE_TESTS_GLOB = 'test/functional/fixtures/compiler-service/test.js';
-const LEGACY_TESTS_GLOB           = 'test/functional/legacy-fixtures/**/test.js';
-const BASIC_TESTS_GLOB            = 'test/functional/fixtures/**/test.js';
+const MULTIPLE_WINDOWS_TESTS_GLOB      = 'test/functional/fixtures/multiple-windows/test.js';
+// TODO: Find out how to run these tests in headed browser in both, proxy and proxyless modes
```

Comment:
Lets create a separate issue for this and disscuss it on the next daily. I'm not sure that this "TODOs" planning works as expected =)

Reply:
Did it: https://github.com/DevExpress/testcafe-private/issues/112